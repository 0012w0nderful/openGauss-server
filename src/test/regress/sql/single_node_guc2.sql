-- test unsupported guc value
show replconninfo1;
set replconninfo5='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
set replconninfo6='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
set replconninfo7='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
show replconninfo5;
show replconninfo6;
show replconninfo7;