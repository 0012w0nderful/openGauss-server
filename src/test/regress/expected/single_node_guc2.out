-- test unsupported guc value
show replconninfo1;
 replconninfo1 
---------------
 
(1 row)

set replconninfo5='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
ERROR:  unrecognized configuration parameter "replconninfo5"
set replconninfo6='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
ERROR:  unrecognized configuration parameter "replconninfo6"
set replconninfo7='localhost=127.0.0.1 localport=10400 localheartbeatport=10402 localservice=10401 remotehost=127.0.0.1 remoteport=10412 remoteheartbeatport=10414 remoteservice=10413';
ERROR:  unrecognized configuration parameter "replconninfo7"
show replconninfo5;
ERROR:  unrecognized configuration parameter "replconninfo5"
show replconninfo6;
ERROR:  unrecognized configuration parameter "replconninfo6"
show replconninfo7;
ERROR:  unrecognized configuration parameter "replconninfo7"
